tests/profiling/test_nogevent.py

```diff
@@ -0,0 +1,12 @@
+import gevent
+import pytest
+import six
+
+
+TESTING_GEVENT = os.getenv("DD_PROFILE_TEST_GEVENT", False)
+
+
+@pytest.mark.skipif(not TESTING_GEVENT or six.PY3, reason="Not testing gevent on Python 2")
```

Comment:
```suggestion
@pytest.mark.skipif(not TESTING_GEVENT or six.PY3, reason="Not testing gevent on Python 3")
```

Reply:
I always get tripped up with double negatives... the reason for the `skipif` here is that we are not testing gevent on Python 2, which means either we are not testing gevent or we are on python 3. That compute?

Reply:
I read it as "skip if Python 3"

also, the bug only exists on Py2 right? so why would we test on Python 3 only?